.github/workflows/test.yml

```diff
@@ -32,14 +32,15 @@ jobs:
   subgraph-test:
     if: github.event_name == 'push'
     runs-on: ubuntu-latest
-    working-directory: ./subgraph
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
           node-version: 16.x
       - name: Install subgraph dependencies
+        working-directory: ./subgraph
```

Comment:
💚. There are some test that need to be removed (written ~10 months ago and never passed or ran in our CI) so this job will fails until then.

Reply:
The only test that I know work are from `RewardsDistributor`.